alma/lib/ShareOfCheckout/ShareOfCheckoutHelper.php

```diff
@@ -74,7 +74,10 @@ public function __construct(
     public function shareDays()
     {
         $shareOfCheckoutEnabledDate = $this->getEnabledDate();
-        if (!Settings::canShareOfCheckout() || empty($shareOfCheckoutEnabledDate)) {
+        if (
+            Settings::canShareOfCheckout() == ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO
```

Comment:
```suggestion
            Settings::getShareOfCheckoutStatus() === ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO
```

Reply:
@syjust-alma I am afraid that the '===' doesn't match with '0' value. Can you confirm ?

Reply:
@syjust-alma Ok I made check and both return a string. I added this :)